README.md

```diff
@@ -1199,6 +1199,49 @@ interface Config {
 }
 ```
 
+case of Array, you can create a read-only array by using `ReadonlyArray<T>`.
+do not allow changes such as `push()` and `fill()`, but can use features such as `concat()` and `slice()` that do not change the value.
+
+```ts
+const array: ReadonlyArray<number> = [ 1, 3, 5 ];
+```
+
+declaring read-only arguments in [TypeScript 3.4 is a bit easier](https://github.com/microsoft/TypeScript/wiki/What's-new-in-TypeScript#improvements-for-readonlyarray-and-readonly-tuples).
+
+```ts
+function hoge(args: readonly string[]) {
+  args.push(1); // error
+}
+```
+
+TypeScript 3.4 introduces a new construct for literal values called [const assertions](https://github.com/microsoft/TypeScript/wiki/What's-new-in-TypeScript#const-assertions).
```

Comment:
I'd suggest that you rephrase it to something like: `Prefer [const assertions](https://github.com/microsoft/TypeScript/wiki/What's-new-in-TypeScript#const-assertions) for literal values.` . No need to mention TS version, that can be checked in the link. This guide should focus on principles and we should not make it a TypeScript tutorial.